src/einsteinpy/symbolic/ricci.py

```diff
@@ -96,3 +154,39 @@ def symbols(self):
 
         """
         return self.syms
+
+
+class RicciScalar:
+    """
+    Class for defining Ricci Scalar
+    """
+
+    def __init__(self, expression, syms, parent_metric=None):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        expression : ~sympy expression
+            Simple sympy
```

Comment:
```
expression : ~sympy.core.expr.Expr
    Any sympy expression
```

Reply:
Thanks! I used the function type( ) but I could not define the expression type correctly